### PR TITLE
Update hydro.md

### PR DESCRIPTION
**Pull request by undefined-moe**

### CODE REVIEW

**Comment by undefined-moe** [comment]
ping

File: docs/project/hydro.md [hunk 1]
```diff
@@ -14,11 +14,12 @@ LANG=zh . <(curl https://hydro.ac/setup.sh)
 
 使 Hydro 接入洛谷开放平台，您需要进行以下操作：
 
-1. 使用 `hydrooj install @hydrooj/vjudge && hydrooj install https://cdn.luogu.com.cn/lgapi/hydro/hydroplugin-luogu-0.0.1.tgz` 安装相关模块。
-2. 使用 `hydrooj cli luogu importProblem` 导入题库。
-3. 使用 `hydrooj cli luogu addAccount <token>` 设置账号。
-4. 重启 Hydro 进程（`pm2 restart hydrooj`）。
-5. 大功告成！
+1. 使用 `hydrooj install @hydrooj/vjudge` 安装 vjudge 插件（如果之前已安装请跳过该步骤） 
+2. 使用 `hydrooj install https://cdn.luogu.com.cn/lgapi/hydro/hydroplugin-luogu-0.0.1.tgz` 安装题库导入工具。
```
Comment (pandadtdyy) [inline docs/project/hydro.md]:
```suggestion
2. 使用 `hydrooj install https://cdn.luogu.com.cn/lgapi/hydro/hydroplugin-luogu-latest.tgz` 安装题库导入工具。
```

**Review by pandadtdyy — CHANGES_REQUESTED** [review]

File: docs/project/hydro.md (hunk 1 above)
Comment (undefined-moe) [reply docs/project/hydro.md]: https://github.com/hydro-dev/Hydro/blob/master/packages/hydrooj/src/commands/install.ts#L48

Comment (pandadtdyy) [reply docs/project/hydro.md]: Now it supported.

**Review by moesoha — APPROVED** [review]

**Pull request merged**